### PR TITLE
Add natural resources

### PR DESCRIPTION
**Pull request by pabloarosado**
* Add input data (downloaded from [EIA](https://www.eia.gov/international/data/world)).
* Add script to generate yearly/monthly data on natural gas/coal/oil production/consumption/imports/exports/reserves.
* Add file with country standardized names for the EIA datasets.
* Add folder with generated yearly/monthly data.
* Add script to run sanity checks on the data.
* Edit sanity_checks module (currently stored in the `cait` folder, although soon this module may be moved elsewhere).

### CODE REVIEW

File: natural_resources/src/main.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+def transform():
+    pass
+
+
+def load():
+    pass
+
+
+def main():
```
Comment (edomt) [inline natural_resources/src/main.py:22]: Was there a rationale behind leaving `transform` and `load` here?

Comment (pabloarosado) [reply natural_resources/src/main.py:22]: Hi @edomt, I simply copied the format from the climate_change folder. I thought we would add more datasets on natural resources, that would be merged here before producing a final dataset (to be stored in the 'output' folder). If for the moment that's not the plan, I can delete these redundant lines.

Comment (edomt) [reply natural_resources/src/main.py:22]: But my sense was that @HannahRitchie wanted to already build a first small explorer just based on the EIA data – in which case there needs to be already some files in `output` to use in that explorer? (But maybe I misunderstood and she wants to bring in more data from other sources before creating a final file)

Comment (pabloarosado) [reply natural_resources/src/main.py:22]: I think you are right, however, the files she's using on the explorer come from the 'ready' folder. I can either:
1. Move the data from 'ready' to 'output' (if we believe we are not going to add anything new for the time being).
2. Copy the data from 'ready' to 'output' (if we believe that we will end up adding more resources apart from fossil fuels).

Comment (HannahRitchie) [reply natural_resources/src/main.py:22]: Yes, later we will add more metrics to this. But our plan is to first publish (hopefully tomorrow) with EIA fossil fuel data.

For this, all I need is a link to a csv (which I have). The location of this is not relevant to me, so wherever makes sense on the side of the data team.

**Review by HannahRitchie — APPROVED** [review]

File: natural_resources/src/main.py (hunk 1 above)
Comment (edomt) [reply natural_resources/src/main.py:22]: Let's keep it this way for now - for now it doesn't make much sense to write a merging step based only on one source anyway 🤷🏻‍♂️

**Pull request merged**